src/GraphBuilder.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+const rdf = require('rdf-ext')
+
+/**
+ * Builds RDF graphs from the Redux state
+ */
+export default class GraphBuilder {
+  /**
+   * @param {Object} state the Redux state
+   */
+  constructor(state) {
+    this.state = state
+  }
+
+  /**
+   * @return {Graph} an RDF graph that represents the data in the state
+   */
+  get graph() {
+    const dataset = rdf.dataset()
+
+    for (const key in this.state) {
+      /*
+       * TODO: this is just a simple statement insert to show this works. We'll
+       *       want to add more here.
+       */
+      dataset.add(rdf.quad(rdf.namedNode(''),
```

Comment:
Following on to the comments about the outdated `src/rdf/graph.js`... The base rdf library seems to use quads as the default, and then requires parsers to translate into JSON-LD or other serializations, so at this stage we don't really need to worry about storing JSON-LD because we are just displaying it in the modal. Also, quads will be one of the serializations that the catalogers will want to see. 👍